.github/workflows/PRLabeling.yml

```
@@ -15,22 +15,77 @@ jobs:
-        uses: actions-ecosystem/action-add-labels@v1
-        with:
-          labels: ${{ steps.pr_details.outputs.label }}
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          # Save label and color to environment variables
+          echo "LABEL=$LABEL" >> $GITHUB_ENV
+          echo "COLOR=$COLOR" >> $GITHUB_ENV
+
+      - name: Get current labels on PR
+        id: get_labels
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:8:32: Double quote to prevent globbing and word splitting

(shellcheck)